pkg/gardenlet/controller/managedseed/managedseed_actuator.go

```diff
@@ -207,31 +230,38 @@ func (a *actuator) Delete(ctx context.Context, ms *seedmanagementv1alpha1.Manage
 
 	if seed != nil {
 		if seed.DeletionTimestamp == nil {
-			a.deletingInfoEventf(ms, "Deleting seed %s", ms.Name)
+			log.Info("Deleting seed", "seed", client.ObjectKey{Name: ms.Name})
```

Comment:
Let's make this consistently `seed` or `seedName` in all similar log statements.